specifications/api/swagger.yaml

```diff
@@ -2076,13 +2081,15 @@ components:
         - account_index
         - address_pool_gap
         - payment_script_template
+        - status
       properties:
         id: *walletId
         name: *walletName
         account_index: *derivationSegment
         address_pool_gap: *walletAddressPoolGap
         payment_script_template: *scriptTemplate
         delegation_script_template: *scriptTemplate
+        status: *statusIncomplete
```

Comment:
As far as I see the status of complete/active wallet is displayed as:
```
  "state": {
    "status": "syncing",
   ...
  }

or

  "state": {
    "status": "ready"
  }
```

Whereas incomplete wallet's status is not wrapped into `state` and displays only:

```
"status": "incomplete"
```

Maybe it would be nice to keep this convention and have:

```
  "state": {
    "status": "incomplete"
  }
```
(That's something I'd expect as an API user at least)

Reply:
done